examples/avmnist_multimodal/model.py

```diff
@@ -0,0 +1,52 @@
+"""
+Define the Multimodal Deep Learning (MMDL) model and configure model hyperparameters.
+References: 1. https://github.com/pliang279/MultiBench/blob/main/examples/multimedia/avmnist_simple_late_fusion.py
+            2. https://github.com/pliang279/MultiBench/blob/main/examples/multimedia/avmnist_low_rank_tensor.py
+            3. https://github.com/pliang279/MultiBench/blob/main/examples/multimedia/avmnist_multi_interac_matrix.py
+"""
+
+from kale.embed.feature_fusion import BimodalMatrixFusionInteractor, Concat, LowRankTensorFusion
+from kale.embed.image_cnn import LeNet
+from kale.pipeline.base_nn_trainer import MultimodalTrainer
+from kale.pipeline.multimodal_deep_learning import MultimodalDeepLearning
```

Comment:
multimodal_deep_learning or MultimodalDeepLearning is not a good name, inconsistent with other names in pipeline. Maybe multimodal_dl_trainer. Discuss with Shuo/Xianyuan today.

Reply:
`multimodal_trainer` may be better

Reply:
fixed